components/tekton-ci/base/appstudio-pipelines-runner-rolebinding.yaml

```diff
@@ -10,3 +10,16 @@ subjects:
 - kind: ServiceAccount
   name: appstudio-pipeline
   namespace: tekton-ci
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: appstudio-pipelines-namespace-creator-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: appstudio-pipelines-namespace-creator
+subjects:
+- kind: ServiceAccount
+  name: appstudio-pipeline
```

Comment:
I don't think that build pipeline should be able to create namespaces.

Reply:
I wonder though, even if removed here - if all `appstodio-pipeline` SA are gonna get created with the ability to create namespaces, wouldn't it give it this permission anyway? or am I missing something